addons/mail/static/src/components/discuss/discuss.xml

```diff
@@ -30,7 +30,7 @@
         <t t-if="discussView.mobileAddItemHeaderAutocompleteInputView">
             <div class="o_Discuss_mobileAddItemHeader border-bottom">
                 <AutocompleteInput
-                    className="'o_Discuss_mobileAddItemHeaderInput'"
+                    className="'o_Discuss_mobileAddItemHeaderInput flex-grow-1 p-2 mb-3 rounded border'"
```

Comment:
After thinking about it, it would be better to add the `form-control` class directly in the `autocomplete_input` component 🤔 .
What so you think @stefanorigano ?

Reply:
Yes, I agree. I think the[ conclusions we come up ](https://github.com/odoo/odoo/pull/88710#discussion_r854168544)with @alexkuhn are valid in all `AutocompleteInput` occurrences.
Adding the class by default will also ensure consistency and will ease the job of using a specific custom class when the time will come.

Reply:
This change is made here : https://github.com/odoo/odoo/pull/91081